tests/libs/datasets/timeseries_test.py

```diff
@@ -584,7 +584,8 @@ def test_write_read_wide_dates_csv_compare_literal(tmpdir):
 
     # Compare written file with a string literal so a test fails if something changes in how the
     # file is written. The literal contains spaces to align the columns in the source.
-    assert pointer.path_wide_dates().read_text() == (
+    wide_dates_data = pd.read_csv(pointer.path_wide_dates(), dtype=object).to_csv(index=False)
```

Comment:
I think the comment above this is now incorrect, but given the comment I think maybe this test was intentionally not parsing the CSV with pandas, and so I think we should either:
1. Remove this test if we don't think it's valuable.
2. Read the file and gunzip it, but don't parse it through pandas to preserve the original intention of the test.

I'm fine with either one.

Reply:
ah, yeah. I think I'll remove it if we're okay with that 🙏